front/src/applications/operationalStudies/components/MacroEditor/osrdToNge.ts

```diff
@@ -265,7 +271,7 @@ const getNgeTrainruns = (state: MacroEditorState, labels: LabelDto[]) =>
       return {
         id: index + 1,
         name: timetableItem.train_name,
-        categoryId: DEFAULT_TRAINRUN_CATEGORY.id,
+        categoryId: getTrainrunCategoryId(timetableItem.category ?? undefined),
```

Comment:
`timetableItem.category ?? undefined` the nullish operand ?? return the left value if it's not null or undefined, else it return the right side value. So it is not necessary because if the left side is undefined, it is already. 
So is the goal is to return undefined if the value of the left side is null, you should test 

`timetableItem.category === null ? undefined : timetableItem.category`

Reply:
if not you can just use `timetableItem.category`